app/code/Magento/Ui/Component/Filters/Type/AbstractFilter.php

```diff
@@ -63,13 +68,34 @@ public function __construct(
         UiComponentFactory $uiComponentFactory,
         FilterBuilder $filterBuilder,
         FilterModifier $filterModifier,
+        BookmarkManagementInterface $bookmarkManagement,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.4/contributor-guide/backward-compatible-development/)